CMakeLists.txt

```diff
@@ -214,7 +214,7 @@ install(FILES "scap-workbench.appdata.xml"
 if (ASCIIDOCTOR_EXECUTABLE)
     file(GLOB USER_MANUAL_SCREENSHOTS "${CMAKE_CURRENT_SOURCE_DIR}/doc/user_manual/*.png")
     add_custom_command(
-        OUTPUT doc/user_manual.html
+        OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/doc/user_manual.html
```

Comment:
Why do we copy the user manual from `${CMAKE_CURRENT_BINARY_DIR}/doc/` to `${CMAKE_CURRENT_SOURCE_DIR}/doc/` (line 220) just to install from `${CMAKE_CURRENT_SOURCE_DIR}/doc`?

I think it makes sense to output to, and install from `CMAKE_CURRENT_BINARY_DIR`.